.env.template

```diff
@@ -0,0 +1,9 @@
+MYSQLDB_USER=${MYSQLDB_USER}
```

Comment:
this doesn't make much sense. It's kinda a recursion :) Better remove everything after = altogether